src/js/populate_question_form.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+/* global wordle_globals */
+
+var inverted_wordles = {};
+
+// Populate the data on the answer question page
+inverted_wordles.setFormData = function (response, options) {
+    response.json().then(questionFile => {
+        inverted_wordles.setEscapedContent(options.selectors.questionTitle, questionFile.title);
+        inverted_wordles.setEscapedContent(options.selectors.entryMaxLength, questionFile.entryMaxLength);
+
+        let entriesHtml = "";
+        // Refer to https://stackoverflow.com/questions/30452263/is-there-a-mechanism-to-loop-x-times-in-es6-ecmascript-6-without-mutable-varia#answer-37417004
+        // for iterating a given number of times.
+        [...Array(questionFile.entries)].map((_, i) => {
```

Comment:
"map" is misleading with the side-effect. Better to return the individual entry's HTML from the map and then assign the resulting HTML using entriesHtml = entries.join("\n");